src/v3/src/components/Widget/style.scss

```diff
@@ -279,6 +279,10 @@
   }
 }
 
+.nowrap {
```

Comment:
added a new class to style.css after discussion with @glenfannin-okta 
he mentioned that for my use case this would be the proper place to put it and that the overrides approach wouldn't work